WordPress/src/main/java/org/wordpress/android/ui/stats/StatsPublicizeFragment.java

```diff
@@ -97,7 +101,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
             holder.totalsTextView.setText(FormatUtils.formatDecimal(currentRowData.getTotals()));
 
             // image
-            holder.networkImageView.setImageUrl(getServiceImage(serviceName), WPNetworkImageView.ImageType.STATS_SITE_AVATAR);
+            holder.networkImageView.setImageUrl(getServiceImage(serviceName), WPNetworkImageView.ImageType.BLAVATAR);
```

Comment:
I think here is better to call GravatarUtils.fixGravatarUrl with the default picture set to STATUS_404.